<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>536</width>
    <height>592</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Noble Ape</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QGraphicsView" name="graphicsView">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>512</width>
        <height>512</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>512</width>
        <height>512</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>536</width>
     <height>17</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="separator"/>
    <addaction name="actionOpenScript"/>
    <addaction name="actionOpen"/>
    <addaction name="separator"/>
    <addaction name="actionSave"/>
    <addaction name="actionSaveAs"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionViewMap"/>
    <addaction name="actionViewTerrain"/>
    <addaction name="actionViewIdeosphere"/>
    <addaction name="actionViewBraincode"/>
    <addaction name="actionViewGenepool"/>
    <addaction name="actionViewHonor"/>
    <addaction name="actionViewPathogens"/>
    <addaction name="actionViewRelationships"/>
    <addaction name="actionViewPreferences"/>
    <addaction name="actionViewPhasespace"/>
    <addaction name="actionViewVascular"/>
    <addaction name="actionViewSocialsim"/>
    <addaction name="actionViewMeetPlaces"/>
   </widget>
   <widget class="QMenu" name="menuWindow">
    <property name="title">
     <string>Control</string>
    </property>
    <addaction name="actionPause"/>
    <addaction name="separator"/>
    <addaction name="actionNext"/>
    <addaction name="actionPrevious"/>
    <addaction name="separator"/>
    <addaction name="actionTerritory"/>
    <addaction name="actionWeather"/>
    <addaction name="actionShowBrain"/>
    <addaction name="actionShowBraincode"/>
    <addaction name="separator"/>
    <addaction name="separator"/>
    <addaction name="actionFlood"/>
    <addaction name="actionHealthyCarrier"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuView"/>
   <addaction name="menuWindow"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionViewMap">
   <property name="text">
    <string>Map</string>
   </property>
  </action>
  <action name="actionViewTerrain">
   <property name="text">
    <string>Terrain</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionSaveAs">
   <property name="text">
    <string>Save As...</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionOpenScript">
   <property name="text">
    <string>Open Script</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionPause">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Pause</string>
   </property>
  </action>
  <action name="actionNext">
   <property name="text">
    <string>Next Ape</string>
   </property>
  </action>
  <action name="actionPrevious">
   <property name="text">
    <string>Previous Ape</string>
   </property>
  </action>
  <action name="actionTerritory">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Territory</string>
   </property>
  </action>
  <action name="actionWeather">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Weather</string>
   </property>
  </action>
  <action name="actionShowBrain">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show Brain</string>
   </property>
  </action>
  <action name="actionShowBraincode">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Braincode</string>
   </property>
  </action>
  <action name="actionFlood">
   <property name="text">
    <string>Flood</string>
   </property>
  </action>
  <action name="actionHealthyCarrier">
   <property name="text">
    <string>Healthy Carrier</string>
   </property>
  </action>
  <action name="actionViewIdeosphere">
   <property name="text">
    <string>Ideosphere</string>
   </property>
  </action>
  <action name="actionViewBraincode">
   <property name="text">
    <string>Braincode</string>
   </property>
  </action>
  <action name="actionViewGenepool">
   <property name="text">
    <string>Gene Pool</string>
   </property>
  </action>
  <action name="actionViewHonor">
   <property name="text">
    <string>Honor Distribution</string>
   </property>
  </action>
  <action name="actionViewPathogens">
   <property name="text">
    <string>Population Immunity</string>
   </property>
  </action>
  <action name="actionViewRelationships">
   <property name="text">
    <string>Relationship Matrix</string>
   </property>
  </action>
  <action name="actionViewPreferences">
   <property name="text">
    <string>Preferences</string>
   </property>
  </action>
  <action name="actionViewPhasespace">
   <property name="text">
    <string>Phasespace</string>
   </property>
  </action>
  <action name="actionViewVascular">
   <property name="text">
    <string>Vascular</string>
   </property>
  </action>
  <action name="actionViewSocialsim">
   <property name="text">
    <string>Socialsim</string>
   </property>
  </action>
  <action name="actionViewMeetPlaces">
   <property name="text">
    <string>Meeting Places</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
